eclipse-scout-core/src/testing/JasmineScoutUtil.ts

```diff
@@ -100,6 +102,18 @@ export const JasmineScoutUtil = {
     });
   },
 
+  mockDataObjectRestCall(resourceUrlToMock: string, responseData: BaseDoEntity | void, options: {
+    restriction?: any;
+    method?: string;
+  } = {}) {
+    let url = new RegExp('.*' + strings.quote(resourceUrlToMock) + '.*');
+    let data = options.restriction ? new RegExp('.*' + strings.quote(options.restriction) + '.*') : undefined;
+    jasmine.Ajax.stubRequest(url, data, options.method).andReturn({
```

Comment:
Please remove duplicated code. E.g. improve mockRestCall so that responseData may be a function that returns the responseText. By default it uses JSON.stringify() and mockDataObjectRestCall may pass dataobjects.stringify(responseData)